tests/e2e/smoketest-simple/01-assert.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: tempo.grafana.com/v1alpha1
+kind: Microservices
+metadata:
+  name: simplest
+---
```

Comment:
Could you please assert:
* services
* exposed ports
* labels on the object meta
* match labels